sdk/rust/oak/src/http/util.rs

```diff
@@ -0,0 +1,103 @@
+//
```

Comment:
Copied from `oak_runtime/src/node/http/util.rs`, but uses the SDK version of some of the types.